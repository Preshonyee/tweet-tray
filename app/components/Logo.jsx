import React from 'react';
import Styled from 'styled-components';

import LogoImage from '../../resources/tweet-tray-logo.svg';

const LogoStyle = Styled.img`
    width: 34px;
    height: 28px;
    position: relative;
    top: 40px;
`;

const Logo = () => {
  return (
    <LogoStyle src={LogoImage} alt="Twitter Logo" />
  );
};

export default Logo;

