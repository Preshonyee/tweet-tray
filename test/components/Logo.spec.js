import React from 'react';
import Enzyme, { shallow, } from 'enzyme';
import { shallowToJson, } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Logo from '../../app/components/Logo';

Enzyme.configure({ adapter: new Adapter(), });

function setup() {
  const component = shallow(<Logo />);
  return {
    component,
  };
}

describe('Logo component', () => {
  it('should render image', () => {
    const { component, } = setup();
    expect(shallowToJson(component)).toMatchSnapshot();
  });
  it('should render image with alt text', () => {
    const { component, } = setup();
    expect(component.prop('alt')).toMatchSnapshot();
  });
});
