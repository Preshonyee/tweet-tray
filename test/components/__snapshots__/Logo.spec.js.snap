// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should render alt text 1`] = `"Twitter Logo"`;

exports[`Logo component should render image 1`] = `
<Logo__LogoStyle
  alt="Twitter Logo"
  src="test-file-stub"
/>
`;
